src/WebUI/Components/ResourceStatusComponent.tsx

```diff
@@ -15,16 +15,33 @@ export interface IResourceStatusProps extends IVssComponentProperties {
 export class ResourceStatusComponent extends BaseComponent<IResourceStatusProps, {}> {
 
     public render(): React.ReactNode {
+        if (this.props.toolTipText) {
+            return (
+                <div className="kube-status-container">
```

Comment:
you are using same class name twice, also why do you want to define div here rather in tooltip function?